parts/query-grid.html

```diff
@@ -0,0 +1,34 @@
+<!-- wp:query { "query":{ "inherit": true }, "displayLayout": { "type": "flex", "columns":4 } } -->
```

Comment:
I preserved this grid template as it was already in the POC. For now the code will use only the list view.

Reply:
I now included a feature toggle to load this template instead.

Reply:
For now I'm focusing on the list view template, so this one is a bit behind.